docs/snippets/react/override-compose-story-test.compose-story.js.mdx

```diff
@@ -0,0 +1,19 @@
+```js
+// Form.test.js
+
+import { composeStory } from '@storybook/react';
+
+import * as FormStories from './LoginForm.stories';
+
+const FormValidation = composeStory(FormStories.ValidForm, FormStories.default, {
```

Comment:
It might make sense to follow a standard based on the other snippets

```suggestion
import Meta, { ValidForm as ValidFormStory } from './LoginForm.stories';

const FormValidation = composeStory(ValidFormStory, Meta, {
```